cockroachcloud/managing-cmek.md

```diff
@@ -236,11 +236,76 @@ To rotate the CMEK keys for one or more cluster regions:
       --data "@cmek_config.json"
     ```
 
+## Add a region to a multi-region CMEK-enabled cluster
+
+To add a region to a multi-region cluster, update your cluster's region definitions using the CockroachDB Cloud API [Update Cluster](https://www.cockroachlabs.com/docs/api/cloud/v1#operation/CockroachCloud_UpdateCluster) endpoint.
+
+1. Construct the data payload JSON to update you region definitions.
+
+    The payload format depends on which cloud provider you use:
+    - **AWS**:
+        {% include_cached copy-clipboard.html %}
+        ```json
+        # cmek_config.json
+        {
+            "dedicated": {
+                "region_nodes": {
+                    "us-west1": 3,
+                    "us-central1": 5
+                },
+                "cmek_region_specs": [
+                    {
+                        "region": "us-west1",
+                        "key_spec": {
+                            "type": "AWS_KMS",
+                            "uri": "{id-of-key}",
+                            "auth_principal": "{role-with-kms-access}"
+                        }
+                    },
+                    {
+                        "region": "us-central1",
+                        "key_spec": {
+                            "type": "AWS_KMS",
+                            "uri": "{id-of-key-2}",
+                            "auth_principal": "{another-role-with-kms-access}""
+                        }
+                    }
+                ],
+                "hardware": {
+                    "machine_type": "n2-standard-8"
+                }
+            }
+        }
+        ```
+    - **GCP**:
+
+        {% include_cached copy-clipboard.html %}
+        ```json
+
+        ```
+1. Send the request to the update cluster endpoint:
```

Comment:
I can't spot an "update cluster" endpoint. I think the magic is that it's a `PATCH` request, but it's just the cluster's own endpoint.

Reply:
fair point.